src/Asset/TagRenderer.php

```diff
@@ -55,10 +72,28 @@ public function renderWebpackScriptTags(string $entryName, string $packageName =
     public function renderWebpackLinkTags(string $entryName, string $packageName = null, string $entrypointName = '_default'): string
     {
         $scriptTags = [];
-        foreach ($this->getEntrypointLookup($entrypointName)->getCssFiles($entryName) as $filename) {
+        $entryPointLookup = $this->getEntrypointLookup($entrypointName);
+        $integrityHashes = ($entryPointLookup instanceof IntegrityDataProviderInterface) ? $entryPointLookup->getIntegrityData() : [];
+
+        foreach ($entryPointLookup->getCssFiles($entryName) as $filename) {
+            $attributes = [
+                'rel' => 'stylesheet',
+                'href' => $this->getAssetPath($filename, $packageName),
+            ];
+
+            if (!empty($integrityHashes[$filename])) {
+                $attributes['integrity'] = $integrityHashes[$filename];
+            }
+
             $scriptTags[] = sprintf(
-                '<link rel="stylesheet" href="%s">',
-                htmlentities($this->getAssetPath($filename, $packageName))
+                '<link %s>',
+                implode(' ', array_map(
+                    function ($key, $value) {
+                        return sprintf('%s="%s"', $key, htmlentities($value));
+                    },
+                    array_keys($attributes),
+                    $attributes
+                ))
```

Comment:
It might be nice to put this logic into a private function - like `convertArrayToAttributes`